airbyte_cdk/sources/declarative/requesters/query_properties/property_chunking.py

```diff
@@ -52,8 +52,9 @@ def get_request_property_chunks(
         chunk_size = 0
         for property_field in property_fields:
             # If property_limit_type is not defined, we default to property_count which is just an incrementing count
+            # +2 for the comma ToDo: Add possibility to specify parameter representation and take it into account in property_field_size
             property_field_size = (
-                len(property_field)
+                len(property_field) + 2
```

Comment:
nit, @tolik0 i think can actually just be one. Because the delimiter in https://github.com/airbytehq/airbyte-python-cdk/blob/main/airbyte_cdk/sources/declarative/requesters/request_options/interpolated_request_options_provider.py#L105-L107 is actually just one `,` and not a space. So 1 is probably sufficient. 

If we still run into request URL issues during the migration, we can always make it smaller than 15000, just to buffer some space for the rest of the non property chunked URL like the `https://api.hubspot...etc`. I'll change this to +1 and adjust the comment and any tests.